src/pages/add-user.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { SafeAreaView, ScrollView, Text, TouchableOpacity, View } from 'react-native';
+import TextInputComponent from '../components/TextInput';
+import { general } from '../styles';
+import TextInputMask from 'react-native-text-input-mask';
+
+const AddUser = () => {
+  const [role, setRole] = React.useState('user');
+  const [date, setDate] = React.useState('');
+  const handleDateChange = (formatted: string) => {
+    const formattedArray = formatted.split('/');
+    if (eval(formattedArray[0]) > 31) {
```

Comment:
`eval` não é uma função meio perigosa, ela executa a string como um script, acredito que somente em casos muito específicos será necessário o uso do `eval` ([página do eval no mdn web docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval)). Pelo que eu entendi vc quer passar uma string pra numero, nesse caso tem outras formas, por exemplo:

```suggestion
    if (+formattedArray[0] > 31) {
```
```suggestion
    if (Number(formattedArray[0]) > 31) {
```